articles/references/concepts/stereotypes.md

```diff
@@ -0,0 +1,76 @@
+
+## How do I get Stereotypes?
+
+Stereotypes are typically created by Module creators, so when you install a new Module for Intent Architect, it typically comes bundled with a set of Stereotypes that a developer can use to describe their application in a richer fashion.
+
+You may however create your own Stereotype Definition from within Intent Architect. Please note however, it will not be used by any of the Modules that you have installed in order to generate any source code. You may however like to add some comments or indicators with the use of Stereotypes that would give developers some visual and informational cues as to what to do or understand about the model they are working in.
+
+![New Stereotype Definition](images/stereotypes/new-stereotype-definition.png)
+
+To create a custom Stereotype, create a new Stereotype definition.
+
+![New Stereotype Definition Role](images/stereotypes/new-stereotype-definition-role.png)
+
+![Properties](images/stereotypes/new-stereotype-definition-role-properties.png)
+
+Name|Description
+-|-
+Name|Name of your Stereotype
+Targets|Which elements are the Stereotype applicable for?
+Icon|A nice visual representation of your Stereotype
+Display Icon|Should the icon be displayed in the modeler?
+Auto Add|Should this Stereotype be automatically added on the creation of a new element found in your `Targets` list?
+Display Text|Writes out text on the element where the stereotype is applied and with the use of expressions, display certain information about the stereotype for that element. An example would be for Web API where an operation would need to have a POST/GET/etc verb and adding a `HTTP` stereotype will render on the screen what kind of `HTTP` verb is required.
+Id|This is for debugging and troubleshooting purposes.
+
+
+Notice that this is not all. Stereotypes can also have fields assigned to them.
+
+![Fields](images/stereotypes/new-stereotype-definition-role-properties-fields.png)
+
+|Name|Description|Type Description
+|-|-|-
+|Name|The name of the field|
+|Type|The type of field you're working with.|
+| |Kind of Types that there are:|
+| | * String|Single-line piece of text
+| | * Text|Multi-line text
+| | * Number|An integer number
+| | * Boolean|True / False
+| | * Options|Create your own options that can be selected
+| | * Lookup Type|Specify the type of elements that can be looked up and there will be a dropdown list of all those items
+| | * Lookup Internal|Not sure. TODO
```

Comment:
We should resolve this before merge, @garethbaars, can you just add a comment here on what this is?

Reply:
Yes, would be great